.travis.yml

```diff
@@ -0,0 +1,17 @@
+language: emacs-lisp
+sudo: false
+cache: apt
+env:
+  - EVM_EMACS=emacs-25.1-travis
+  - EVM_EMACS=emacs-25.2-travis
+before_install:
+  - export PATH="/home/travis/.evm/bin:$PATH"
+  - git clone https://github.com/rejeep/evm.git /home/travis/.evm
+  - evm config path /tmp
```

Comment:
Don't think this really matters. EVM says to use `/tmp`, and I don't want to run into issues with that, so I'm just going to leave it.